### DISCUSSION TITLE
relationMappings by one-to-one or one-to-many relationship JSON column

### DISCUSSION THREAD

**Discussion by qansuh** [Vincit/objection.js#2485] [Questions]
Please help solve this issue
I have tried a lot to create a one-to-one or one-to-many relationship between table2 -> table1
, but I did not succeed in finding a solution. The example is as follows

Table1
id   ,  name 
------------
1   ,  name
2   ,  name
3   ,  name

================

Table2
id  ,  name                          , options (json)
---------------------------------------
1  ,  name              { "table1": { "ids": ["1", "2"] } }
2  ,  name              { "table1": { "ids": ["3", "1"] } }

[https://github.com/staudenmeir/eloquent-json-relations#many-to-many-relationships](url)

What I want is something similar, how can I create this kind of relation with Objection.js?

**Comment by lehni** [comment]
I've moved this to discussions, because it's not a bug. 

Have you searched previous questions / seen this? I think it should answer your question.

https://github.com/Vincit/objection.js/issues/1956
